crashDataAnalysisTools/plot_and_save.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pandas as pd
+from matplotlib.backends.backend_pdf import PdfPages
+
+
+def PlotXVsY(df, colname_x, colname_y):
+    """
+    scatter plot of attribute x against y
+    Parameters:
+    @df {dataframe} datasource
+    @colname_x {string} column name for attribute x
+    @colname_y {string} column name for attribute y
+    Return:
+    the produced figure
+    """
+    plt.scatter(df[colname_x], df[colname_y])
+    plt.xlabel(colname_x)
+    plt.ylabel(colname_y)
+    plt.show()
+    return plt.figure()
+
+
+def PlotModelCDF(rmodel, lowerb, upperb, modelname):
+    """
+    plot of model CDF
```

Comment:
It seems that the two input variables lowerb and upperb are never used in the function. Maybe add some more introduction about how the function works with the inputs?